src/main/kotlin/app/cash/licensee/licenses.kt

```diff
@@ -65,28 +65,54 @@ private fun PomLicense.toSpdxOrNull(): SpdxLicense? {
       "http://www.apache.org/licenses/LICENSE-2.0.html",
       "https://www.apache.org/licenses/LICENSE-2.0.html",
       "http://www.opensource.org/licenses/apache2.0.php",
+      "https://spdx.org/licenses/Apache-2.0",
+      "https://aws.amazon.com/apache2.0",
       -> "Apache-2.0"
 
       "http://creativecommons.org/publicdomain/zero/1.0/",
+      "https://creativecommons.org/publicdomain/zero/1.0/",
       -> "CC0-1.0"
 
       "http://www.opensource.org/licenses/LGPL-2.1",
+      "http://www.gnu.org/licenses/lgpl-2.1.html",
+      "https://www.gnu.org/licenses/old-licenses/lgpl-2.1",
       -> "LGPL-2.1-only"
 
+      "http://www.gnu.org/licenses/lgpl-3.0.txt",
+      -> "LGPL-3.0-only"
+
       "https://opensource.org/licenses/mit-license",
       "http://www.opensource.org/licenses/mit-license.php",
       "http://opensource.org/licenses/MIT",
       -> "MIT"
 
       "http://www.opensource.org/licenses/bsd-license",
+      "http://www.opensource.org/licenses/bsd-license.php",
       -> "BSD-2-Clause"
 
+      "http://www.eclipse.org/org/documents/edl-v10.php",
+      -> "BSD-3-Clause"
```

Comment:
For reference https://lists.spdx.org/g/Spdx-legal/topic/request_for_adding_eclipse/67981884